.github/workflows/patch-code-of-conduct.yml

```diff
@@ -0,0 +1,26 @@
+name: Patch CoC
+on: [push]
+
+jobs:
+  patch:
+    runs-on: ubuntu-latest
+    name: Patch
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          persist-credentials: false
+      - name: Patch CoC
+        uses: pkgjs/patch-my-code-of-conduct@v1.1.0
+        with:
+          base_url: 'https://raw.githubusercontent.com/openjs-foundation/cross-project-council/9a4beb1593f6156cc83dc83038f7af65d8a34971/conduct/artifacts/CODE_OF_CONDUCT_PREAMBLE.md'
```

Comment:
is there a reason we don't want this to run on HEAD?

Reply:
Referencing your previous comment:

> Isn’t the point of the base that it’s a 1:1 copy of what’s in the upstream? meaning the base shouldn’t even need to be committed, since it can be fetched by sha

The initial request was to have an SHA that points to a commit. That's why I removed the relative path, and added this URL based fetching solution.

Reply:
ahh - i meant the *upstream COC* by SHA. the preamble seems fine to me to just use latest.

Reply:
I can always rollback, and do the initial implementation which is a lot easier to maintain (no need to commit and later change workflow config.)

Reply:
Sorry for being unclear and the extra work :-/

Reply:
No worries at all. I'll rollback

Reply:
Can you change your review to `Request change` to block it?